benchmarks/sparse/KokkosSparse_par_ilut.cpp

```diff
@@ -41,24 +44,147 @@ using KokkosSparse::spiluk_symbolic;
 using KokkosSparse::Experimental::SPILUKAlgorithm;
 
 // Build up useful types
-using scalar_t  = KokkosKernels::default_scalar;
-using lno_t     = KokkosKernels::default_lno_t;
-using size_type = KokkosKernels::default_size_type;
-using exe_space = Kokkos::DefaultExecutionSpace;
-using mem_space = typename exe_space::memory_space;
-using device    = Kokkos::Device<exe_space, mem_space>;
-
-using RowMapType  = Kokkos::View<size_type*, device>;
-using EntriesType = Kokkos::View<lno_t*, device>;
-using ValuesType  = Kokkos::View<scalar_t*, device>;
-
-using sp_matrix_type = KokkosSparse::CrsMatrix<scalar_t, lno_t, device, void, size_type>;
-using KernelHandle =
-    KokkosKernels::Experimental::KokkosKernelsHandle<size_type, lno_t, scalar_t, exe_space, mem_space, mem_space>;
-using float_t = typename KokkosKernels::ArithTraits<scalar_t>::mag_type;
+using scalar_t    = KokkosKernels::default_scalar;
+using lno_t       = KokkosKernels::default_lno_t;
+using size_type_t = lno_t;  // ginkgo does not have the concept of size_type separate from ordinal type
```

Comment:
I am fine with ```size_type_t```. Is adding ```_t``` to agree with other type names? I just feel it is a bit redundant as ```size_type``` already means a type.